nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-api/src/main/java/org/nuxeo/ecm/platform/ec/notification/NotificationConstants.java

```diff
@@ -83,6 +83,9 @@
 
     public static final String IS_JSF_UI = "isJSFUI";
 
+    /** @since 11.1 **/
```

Comment:
@since 2021.15 ?

Reply:
@guirenard cherry-picked an old commit that was done on `11.1`, not worth updating it IMO, except if @guirenard wants to update to `10.10-HF58` ofc :)